src/components/UserImpact.js

```diff
@@ -49,6 +50,7 @@ const UserImpact = ({ userImpact, user }) => {
     pendingUserReferralImpact,
     pendingUserReferralCount,
   } = userImpact
+  const { currentMission, pendingMissionInvites } = user
```

Comment:
Just noticed that we need to add pendingMissionInvites field to the UserImpactContainer in order for it to make it here